CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ CHANGELOG
 **BUG FIXES**
 - Fix validation of parameter `SharedStorage/EfsSettings`: now validation fails when `FileSystemId` is specified 
   along with other `SharedStorage/EfsSettings` parameters, whereas it was previously ignoring them.
+- Fix cluster update when switching the order of SharedStorage configuration.
```

Comment:
if I'm not wrong we don't support yet an update of this section. In that case this is not really a bug but will be part of adding support for shared FS updates.

Reply:
It's a bug because it's triggering the update of the shared storages when there are no changes on them except for the order. 
This issue is triggered only when there are other changes in the config.

So probably we should add something like:
`...together with other changes in the configuration.`

Reply:
We support update of some of the parameters in SharedStorage section, for example, IOPS, Throughput, DeletionPolicy